### PR TITLE
Sample and description

### PR DESCRIPTION
**Pull request by E-Mani**
Added the sample and description for chart getting started.

### CODE REVIEW

File: Chart_GettingStarted/Chart_GettingStarted/Chart_GettingStarted.Android/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Chart_GettingStarted.Android")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("Chart_GettingStarted.Android")]
+[assembly: AssemblyCopyright("Copyright ©  2014")]
```
Comment (karthikeyanvis) [inline Chart_GettingStarted/Chart_GettingStarted/Chart_GettingStarted.Android/Properties/AssemblyInfo.cs]: @E-Mannan  Add the copyright 2018-2019

File: README.md [hunk 2]
```diff
@@ -1,2 +1,13 @@
...
+For more details please refer the Xamarin.Forms SfChart UG documentation [Getting Started](https://help.syncfusion.com/xamarin/sfchart/getting-started) link.
+
+## <a name="requirements-to-run-the-demo"></a>Requirements to run the demo ##
+
+* [Visual Studio 2017](https://visualstudio.microsoft.com/downloads/) or [Visual Studio for Mac](https://visualstudio.microsoft.com/vs/mac/).
+* Xamarin add-ons for Visual Studio (available via the Visual Studio installer).
+
+## <a name="troubleshooting"></a>Troubleshooting ##
+### Path too long exception
+If you are facing path too long exception when building this example project, close Visual Studio and rename the repository to short and build the project.
```
Comment (karthikeyanvis) [inline README.md:13]: @E-Mannan Add a whole getting started document like in our site. 

https://help.syncfusion.com/xamarin/sfchart/getting-started

File: Chart_GettingStarted/Chart_GettingStarted/Chart_GettingStarted.Android/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (E-Mani) [reply Chart_GettingStarted/Chart_GettingStarted/Chart_GettingStarted.Android/Properties/AssemblyInfo.cs]: @karthikeyanvis Changed

File: README.md (hunk 2 above)
Comment (E-Mani) [reply README.md:13]: @karthikeyanvis added

**Pull request merged**